ctrd/image.go

```diff
@@ -196,8 +195,6 @@ func (c *Client) PullImage(ctx context.Context, ref string, authConfig *types.Au
 }
 
 func (c *Client) pullImage(ctx context.Context, wrapperCli *WrapperClient, ref string, options []containerd.RemoteOpt) (containerd.Image, error) {
-	ctx = leases.WithLease(ctx, wrapperCli.lease.ID())
-
 	img, err := wrapperCli.client.Pull(ctx, ref, options...)
```

Comment:
should double check!!!

Reply:
checked it. The GC doesn't clean your image if you don't remove it.